src/plugins/gc-checklist/gc-checklist-en.hbs

```diff
@@ -0,0 +1,173 @@
+	"breadcrumb": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html" },
+		{ "title": "Provisional functionality", "link": "https://wet-boew.github.io/themes-dist/GCWeb/provisional-en.html" }
+	],
+	"dateModified": "2020-12-31",
+	"description": "Working examples for checklist - Provisional feature",
+	"language": "en",
+	"pageclass": "cnt-wdth-lmtd",
+	"parentdir": "component",
+	"tag": "checklist",
+	"title": "Checklist - Provisional feature"
+}
+---
+{{>alert-provisional-demos}}
+<span class="wb-prettify all-pre hide"></span>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Provisional</dd>
+	<dt>Type</dt>
+	<dd>Canada.ca design pattern</dd>
+	<dt>Last review</dt>
+	<dd>2020-12-31</dd>
+	<dt>Conforming to</dt>
+	<dd>&nbsp;</dd>
+	<dt>Guidance</dt>
+	<dd><a href="https://design.canada.ca/common-design-patterns/checkboxes-radio-buttons.html">Checkboxes and radio buttons - Beta - Canada.ca design pattern</a></dd>
+	<dt>Technical name</dt>
+	<dd>
+		<ul>
+			<li><code>gc-checklist</code></li>
+		</ul>
+	</dd>
+</dl>
+<h2>Working example</h2>
+<section>
+	<h3>Example 1</h3>
+	<p>[To qualify, you must meet <strong>all</strong> of the following required conditions:] <small class="text-muted">(checklist, large boxes)</small></p>
+	<ul class="provisional gc-checklist lst-none lst-spcd-2" data-gc-checklist='{ "feedbackSuccess": "[You are ready to call]", "feedbackFailure": "[You need all of the required criteria to call]", "feedbackTarget": ".feedback span" }' >
+		<li class="checkbox">
+			<input type="checkbox" id="cond1" required="required">
+			<label for="cond1">[Condition 1] (required)</label>
+		</li>
+		<li class="checkbox">
+			<input type="checkbox" id="cond2">
+			<label for="cond2">[Condition 2]</label>
+		</li>
+		<li class="checkbox">
+			<input type="checkbox" id="cond3" required>
+			<label for="cond3">[Condition 3] (required)</label>
+		</li>
+		<li class="feedback"><span class="label label-danger">[You need all of the required criteria to call]</span></li>
+	</ul>
+</section>
+<section>
+	<h3>Example 2</h3>
+	<p>[To qualify, you must meet <strong>all</strong> of the following conditions:] <small class="text-muted">(checklist, small boxes)</small></p>
+	<ul class="provisional gc-checklist lst-none lst-spcd" data-gc-checklist=' { "feedbackSuccess": "[You are ready to call]", "feedbackFailure": "[You need all of the above to call]", "feedbackTarget": ".feedback span" } '>
+		<li class="checkbox-sm">
+			<input type="checkbox" id="cond4" required>
+			<label for="cond4">[Condition 4] (required)</label>
+		</li>
+		<li class="checkbox-sm">
+			<input type="checkbox" id="cond5" required>
+			<label for="cond5">[Condition 5] (required)</label>
+		</li>
+		<li class="checkbox-sm">
+			<input type="checkbox" id="cond6" required>
+			<label for="cond6">[Condition 6] (required)</label>
+		</li>
+		<li class="feedback"><span class="label label-danger">[You need all of the above to call]</span></li>
```

Comment:
This list item are not the same kind compared to it's sibling. Please move it as paragraph after the unordered list.